pkg/kubelet/dockertools/instrumented_docker.go

```diff
@@ -34,130 +34,158 @@ func NewInstrumentedDockerInterface(dockerClient DockerInterface) DockerInterfac
 	}
 }
 
+// Record the duration of the operation.
+func recordOperation(operation string, start time.Time) {
+	metrics.DockerOperationsLatency.WithLabelValues(operation).Observe(metrics.SinceInMicroseconds(start))
+}
+
+// Record error for metric if an error occured.
+func recordError(operation string, err error) {
+	if err != nil {
+		metrics.DockerErrors.WithLabelValues("list_containers").Inc()
```

Comment:
Ah good catch. Fixed.

(So *thats* why I only got errors for list_containers...)